openssl-dynamic/src/main/native-package/m4/tcnative.m4

```diff
@@ -275,6 +275,32 @@ dnl Note: this define must be on one line so that it can be properly returned
 dnl as the help string.
 AC_DEFUN([TCN_HELP_STRING],[ifelse(regexp(AC_ACVERSION, 2\.1), -1, AC_HELP_STRING($1,$2),[  ]$1 substr([                       ],len($1))$2)])dnl
 
+dnl
+dnl TCN_CHECK_STATIC
+dnl Will prepare more LDFLAGS that should be set to ensure we not export any functions from the static compiled APR / OpenSSL libs.
+dnl
+AC_DEFUN([TCN_CHECK_STATIC],[
```

Comment:
I had to do this as an autoconf function as otherwise the flags would have been picked up during running `./configure` and so may fail the test to detect the linker / gcc as the filtered symbols did not exist there yet.